docs/contributers/src/SUMMARY.md

```diff
@@ -9,10 +9,12 @@
     - [Routing Layer](./technical/routing/_intro.md)
         - [Routing Table](./technical/routing/routing-table.md)
         - [Network Modules](./technical/routing/netmod.md)
+	- [API Layer](./technical/api/_intro.md)
+		- [HTTP](./technical/api/http/_intro.md)
     - [`libqaul`](./technical/libqaul/_intro.md)
     - [Web GUI](./technical/webgui/_intro.md)
 - [Translations](./translations/_intro.md)
 - [Design](./design/_intro.md)
 - [Legal](./legal/_intro.md)
     - [GPL License (Version 3)](./legal/gpl-3.0.md)
-    - [Creative Commons Attribution 4.0 International License](./legal/cc-by.md)
\ No newline at end of file
+    - [Creative Commons Attribution 4.0 International License](./legal/cc-by.md)
```

Comment:
What happened with that line? :sweat_smile:

Reply:
?-? i haven't a clue